source/extensions/omni.isaac.lab/omni/isaac/lab/envs/direct_rl_env.py

```diff
@@ -86,7 +86,7 @@ def __init__(self, cfg: DirectRLEnvCfg, render_mode: str | None = None, **kwargs
 
         # set the seed for the environment
         if self.cfg.seed is not None:
-            self.seed(self.cfg.seed)
+            self.cfg.seed = self.seed(self.cfg.seed)
```

Comment:
Isn't the function from Isaac Sim (torch_utils.set_seed) doing this automatically when the seed is -1?

Reply:
it will generate a random seed if seed=-1 and return the new seed, this assigns the new seed to the env cfg

Reply:
got it.